lektor/admin/utils.py

```diff
@@ -1,3 +1,6 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
```

Comment:
Please do not use unicode literals.  I will not merge code that regresses on Python 2 and unicode literals is the best way to achieve that.  Please use explicit `u` prefixes on Python 3.